src/routes/protected.tsx

```diff
@@ -642,6 +643,15 @@ export const protectedRoutes: RouteNode[] = [
               </RootPermissionsFilter>
             ),
           },
+          {
+            path: `${AdminDashboardRoutes.CONFIGURE_AUTO_EXIT}/*`,
```

Comment:
?
```suggestion
            path: AdminDashboardRoutes.CONFIGURE_AUTO_EXIT,
```

Reply:
Ohh okay I didn't realize that. Sorry, I thought they were leftover routes if you had first had the editing on a different page or something. But yeah, I think it's good to not introduce a new routing pattern for a behavior-pattern that we usually implement with state.